RiotSwiftUI/Modules/Common/Avatar/ViewModel/AvatarViewState.swift

```diff
@@ -15,7 +15,7 @@
 //
 
 import Foundation
-
+import UIKit
```

Comment:
UIImage was referenced below so I think should be there yea. For some reason wasn't causing a build error where the file was before. I think when we I added RiotSwiftUI as a target dependency it started to complain. It may still build if I remove it but think it should probably be there.